src/main/java/org/pentaho/di/profiling/datacleaner/ModelerHelper.java

```diff
@@ -118,6 +124,25 @@ public String getName() {
         return "profiler"; //$NON-NLS-1$
     }
 
+    private static String getDataCleanerInstalationPath(String pluginFolderPath) {
+        final String configurationFilePath = pluginFolderPath + "/" + DATACLEANER_CONFIG_FILE;
+
+        try {
+            final BufferedReader inputReader = new BufferedReader(new FileReader(configurationFilePath));
+            final String dcInstallationPath = inputReader.readLine().trim();
+            inputReader.close();
+            return dcInstallationPath;
+        } catch (Exception e) {
+            new ErrorDialog(
```

Comment:
Just wondering/asking: Shouldn't one call _something_ on a dialog to show it? Here it is just instantiated, but I could imagine it having a ".show()" or ".setVisible(true)" like method?